.github/workflows/wheels.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os: [ubuntu-latest, ubuntu-24.04-arm, windows-latest, macos-latest]
```

Comment:
is there a `ubuntu-latest-arm`?

Reply:
nope

Reply:
https://github.com/orgs/community/discussions/148648#discussioncomment-11858287

Doesn't seem to be anything new on that.